Command/GenerateBundleCommand.php

```diff
@@ -350,7 +350,7 @@ protected function updateConfiguration(OutputInterface $output, Bundle $bundle)
             $manipulator->addResource($bundle);
         } catch (\RuntimeException $e) {
             return array(
-                '- Import the bundle\'s %s resource in the app\'s main configuration file:',
+                sprintf('- Import the bundle\'s %s resource in the app\'s main configuration file:', $bundle->getServicesConfigurationFilename()),
```

Comment:
Thanks for submitting this fix :) Can you please surround `%s` with quotes? Then it would be perfect, thank you very much!